main.tf

```diff
@@ -33,8 +35,8 @@ locals {
       message_retention_duration = lookup(subscription, "message_retention_duration", var.default_message_retention_duration)
       expiry_ttl                 = lookup(subscription, "expiry_ttl", local.default_expiry_ttl)
       filter                     = lookup(subscription, "filter", "")
-      minimum_backoff            = lookup(subscription, "minimum_backoff", var.default_minimum_backoff)
-      maximum_backoff            = lookup(subscription, "maximum_backoff", var.default_maximum_backoff)
+      minimum_backoff            = lookup(subscription, "minimum_backoff", local.default_minimum_backoff)
+      maximum_backoff            = lookup(subscription, "maximum_backoff", local.default_maximum_backoff)
```

Comment:
@koshmikumari we don't need to use `locals` here. We are not polishing the value from `var`. So we can simply use `var` directly instead of going through `locals`.